dv/uvm/core_ibex/sim.py

```diff
@@ -404,9 +392,25 @@ def compare_test_run(test, idx, iss, output_dir, report):
 
             return False
 
-        if no_post_compare:
-            report_fd.write('[PASSED]\n')
-            return True
+    rtl_log = os.path.join(rtl_dir, 'trace_core_00000000.log')
+    rtl_csv = os.path.join(rtl_dir, 'trace_core_00000000.csv')
+
+    try:
+        # Convert the RTL log file to a trace CSV.
+        process_ibex_sim_log(rtl_log, rtl_csv, 1)
+    except (OSError, RuntimeError) as e:
+        with open(report, 'a') as report_fd:
+            report_fd.write('[FAILED]: Log processing failed: {}\n'.format(e))
+
+        return False
+
+    no_post_compare = test.get('no_post_compare')
```

Comment:
Nit: It doesn't really matter, but I'd be inclined to do this as `test.get('no_post_compare', False)` to make it clear that this is always supposed to be a boolean. And maybe assert `isinstance(no_post_compare, bool)` too.